### PR TITLE
[Concept Entry]: Add codebyte example - sklearn concepts

### LINKED ISSUE #5334: [Edit] [Concept Entry]: Missing codebyte examples for sklearn entries
### Type of Edit (select all that apply)

Fix grammar, spelling, formatting, Add new content (definitions, codeblocks, etc.)

### Description (optional)

There are multiple `codebyte examples` missing for `sklearn` concept entries. 
https://www.codecademy.com/resources/docs/sklearn

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by saldanhad**
### Description
Added the `codebyte examples` for multiple concept entries under`sklearn`.

### Issue Solved
closes #5334

- Added the `codebyte example` for below algorithms under concept entry for `sklearn`:
    - `Multitask Classification`
    - `Naive Bayes` (added example and codebyte example)
    - `Support Vector Machines`

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by saldanhad** [comment]
Hi @cigar-galaxy82, would we need a second review here or can we merge the changes ?

**Comment by saldanhad** [comment]
any update here?

File: content/sklearn/concepts/naive-bayes/naive-bayes.md
Comment (cigar-galaxy82) [inline content/sklearn/concepts/naive-bayes/naive-bayes.md:54]: remove this blank line

**Comment by cigar-galaxy82** [comment]
Hi @saldanhad thank you for contributing to Codecademy docs, I have a few suggestions for each Codebyte example use a different example code then the one used in the Example section

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @saldanhad once @SaviDahegaonkar reviews for the second time it will get merged

**Comment by saldanhad** [comment]
Hi @cigar-galaxy82, thanks for your reply, still waiting on  #5209 to get merged.

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @saldanhad , 
LGTM! Thanks for contributing to docs.


Thanks,
Savi

**Review by cigar-galaxy82 — APPROVED** [review]

**Pull request merged**
